custom_components/deebot/translations/it.json

```diff
@@ -2,20 +2,37 @@
   "config": {
     "abort": {
       "already_configured": "Già configurato"
+      "reauth_successful": "Ri-autenticazione avvenuta con successo"
     },
     "error": {
-      "cannot_connect": "Non riesco a connettermi con ecovacs API",
-      "invalid_auth": "Username o password errate",
+      "cannot_connect": "Non riesco a connettermi con le API di ecovacs",
+      "invalid_auth": "Utente o password errati",
       "unknown": "Errore sconosciuto",
-      "invalid_country": "Il country code deve essere di due lettere, es: it, us, uk etc ",
-      "invalid_continent": "Il Continent code deve essere di due lettere, es: eu, ww etc ",
-      "select_robots": "Seleziona almeno 1 robot"
+      "invalid_country": "La sigla paese code deve essere di due lettere, es: it, us, uk ecc...",
+      "invalid_continent": "La sigla contiente code deve essere di due lettere, es: eu, ww ecc...",
+      "select_robots": "Seleziona almeno 1 robot",
+      "no_supported_devices_found": "Non sono stati trovati dispositivi supportati"
     },
     "step": {
       "user": {
-        "robots": {
+        "data": {
+          "password": "Password",
```

Comment:
```suggestion
```

Don‘t duplicate translation that are the same in english, it will fallback automatically to en.